src/ui/components/DevToolsProcessingScreen.tsx

```diff
@@ -30,8 +53,21 @@ export function DevToolsProcessingScreen() {
     }
   }, [processingProgress]);
 
+  useEffect(() => {
+    return () => {
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+      const { permission, permissionsRequested } = notificationPermissionStateRef.current;
+      if (permissionsRequested && permission === "granted" && recording) {
+        const progress = getProcessingProgress(store.getState());
+        if (progress === 100) {
```

Comment:
to avoid creating notifications when the user is looking at our app we could combine this with something like:
```suggestion
        if (progress === 100 && !document.hasFocus()) {
```